projects/plugins/jetpack/modules/contact-form/js/form-style.js

```diff
@@ -0,0 +1,52 @@
+jQuery( function ( $ ) {
+	$( document ).ready( function () {
```

Comment:
This will cause issues if the content is dynamically loaded though - P2 theme being a good example to test against. Instead of relying on `$( document ).ready()` I'd suggest using something like `MutationObserver` to find any new dropdown block elements on the page whenever they appear.